README.md

```diff
@@ -1,69 +1,77 @@
 Then, if you use Gradle, add the following to the `build.gradle` file in
-the project you wish to type-check (using Maven is similar):
+the project you wish to type-check:
 
 ```
-repositories {
-    mavenLocal()
-    mavenCentral()
+plugins {
+  id 'org.checkerframework' version '0.6.28'
+  ...
+}
+
+...
+
+apply plugin: 'org.checkerframework'
+checkerFramework {
+  checkers = [
+    'org.checkerframework.checker.dependencyinjection.DependencyInjectionChecker',
+  ]
 }
+
+...
+
 dependencies {
-    annotationProcessor 'org.checkerframework:dependencyinjection-checker:0.1-SNAPSHOT'
+    compileOnly 'org.checkerframework:dependencyinjection-checker-qual:0.1-SNAPSHOT'
+    testCompileOnly 'org.checkerframework:dependencyinjection-checker-qual:0.1-SNAPSHOT'
+    checkerFramework 'org.checkerframework:dependencyinjection-checker:0.1-SNAPSHOT'
+    ...
 }
 ```
 
-Now, when you build your project, the Dependency Injection Checker will also run,
-informing you of any potential errors related to TODO.
+Now, when you build your project, the Guice Checker will also run, informing you of any potential errors related to improperly defined or configured bindings.
 
 
 ## How to specify your code
 
-At compile time, the Dependency InjectionChecker estimates what values the program
-may compute at run time.  It issues a warning if the program may TODO.
-It works via a technique called pluggable typechecking.
+At compile time, the Guice Checker estimates what bindings the program may compute at run time.  It issues a warning if the program attempts to request a binding that has not been properly defined or configured. It works via a technique called pluggable typechecking.
 
-You need to specify the contracts of methods and fields in your code --
-that is, their requirements and their guarantees.  The Dependency InjectionChecker
-ensures that your code is consistent with the contracts, and that the
-contracts guarantee that TODO.
+Most of the time, the Guice Checker will work automatically. However, sometimes you may need to write an annotation to specify dataflow that is relevant to Guice bindings, if it crosses procedure boundaries. You can achieve that with the following type annotations:
 
-You specify your code by writing *qualifiers* such as `@DependencyInjectionBottom`
-on types, to indicate more precisely what values the type represents.
-Here are the type qualifiers that are supported by the Dependency InjectionChecker:
+`@Bind({})`
 
-`@DependencyInjectionUnknown`:
-The value might or might not be TODO. It is not safe to use for TODO.
-This is the default type, so programmers usually do not need to write it.
+- The default qualifier of this system. If an expression's type has this qualifier, then the expression's value may not result from a direct method invocation of `com.google.inject.AbstractModule.bind`. It implies that no arguments have been passed to the `bind` method. Since this is the default type, programmers usually do not need to write it.
```

Comment:
What does the phrase "direct method invocation" mean? Is it different than a method invocation?